RIMdev/wallpapers/README.md

```diff
@@ -38,4 +38,10 @@ All wallpapers are 1920x1080
 ![RIMdev Saved By The Dev](rimdev-saved-by-the-dev-1920x1080.jpg)
 
 ## Indiana Jones
-![Indiana Jones](rimdev-indiana-jones-1920x1080.jpg)
\ No newline at end of file
+![Indiana Jones](rimdev-indiana-jones-1920x1080.jpg)
+
+## RIMdev Marvel
+![RIMdev Saved By The Dev](rimdev-iron-man-1920x1080.jpg)
+![RIMdev Saved By The Dev](rimdev-ant-man-1920x1080.jpg)
+![RIMdev Saved By The Dev](rimdev-spider-man-1920x1080.jpg)
+![RIMdev Saved By The Dev](rimdev-octupus-prime-1920x1080.jpg)
```

Comment:
Mind updating the text descriptions here?

Reply:
Specifically the "RIMDev Saved By The Dev" part to match each image.

Reply:
oh, got it. Thanks, Ken!